themes-default/slim/package.json

```diff
@@ -26,7 +26,9 @@
       }
     }
   },
-  "dependencies": {},
+  "dependencies": {
+    "domready": "1.0.8"
```

Comment:
Added to normal dependencies by mistake?

Reply:
All dependencies that are imported into webpack should really be added to `dependencies` not `devDependencies` since they're used in the app/bundle just not in dev.

The `devDependencies` is meant for build tools, etc.

Reply:
And here I thought I was the only one thinking that :wink: